subgraph/schema.graphql

```diff
@@ -131,13 +131,20 @@ type Transfer @entity(immutable: true) {
 }
 
 type Token @entity {
-  id: Bytes! #transaction hash
+  id: Bytes! # Token ID
+  mint_transaction_hash: Bytes!
   tokenId: BigInt! # uint256
   owner: Holder!
   minted_by: Bytes!
+  controllers: [Controller!]
 }
 
 type Holder @entity {
```

Comment:
Should we change this to Owner to match the contract?

Reply:
controllers are different from owners. Owner refers to the ERC721 owner of the NFA who can burn and transfer the asset. Controllers can only update information.

Reply:
Yes sorry I clicked on the wrong, line, I meant line 142 where we have `Holder`.

Reply:
That's handled in the PR 100.